geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/ConverterConfigResolver.scala

```diff
@@ -33,10 +33,13 @@ object ConverterConfigResolver extends LazyLogging {
   /**
    * @return the converter config parsed from the args
    */
-  def getConfig(configArg: String): Option[Config] =
-    getLoadedConf(configArg)
+  def getConfig(configArg: String): Either[String, Config] = {
+    val conf = getLoadedConf(configArg)
       .orElse(parseFile(configArg))
       .orElse(parseString(configArg))
+    if (conf.isDefined) Right(conf.get)
+    else Left(s"Unable to parse Converter config from argument $configArg")
```

Comment:
this isn't really offering any additional info - maybe instead we'd have a failure message for each thing we tried to do and show them all? Something like:
`"Unable to parse Converter config from argument $configArg. No configuration exists in the environment with that name. No file exists with the name: ${fileError}. Could not parse as a config string: ${stringError}"`
Ideally we'd maybe be able to detect what they are trying (e.g. name, file or string) and report only the correct message.
Same thing with the sft resolver below.